bokeh/charts/_data_adapter.py

```diff
@@ -137,6 +138,13 @@ def validate_values(values):
 
             return values
 
+        elif hasattr(values, '__array__'):
```

Comment:
+1 about removing dependency on `blaze.into`
Yes, both work. My preference is to use `np.asarray()` as it does not copy the object.

Thanks!